pkg/server/home_workspaces.go

```diff
@@ -352,7 +358,8 @@ func (h *homeWorkspaceHandler) ServeHTTP(rw http.ResponseWriter, req *http.Reque
 
 	homeWorkspace := &tenancyv1beta1.Workspace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: this.Name,
+			Name:              this.Name,
+			CreationTimestamp: this.CreationTimestamp,
```

Comment:
rename PR to "SQUASH: Rewrite home workspaces with parent-less workspaces"

Reply:
done